### ISSUE TITLE
Improve "Decline: Do not want"

### ISSUE THREAD

**Issue by Revadike** [bartervg/barter.vg#26]
In an effort to improve offer preferences. Perhaps you could implement a dialog to refine / take another look at their offer preferences, when they select to decline an offer, because they don't want the offered tradable(s). Maybe even add a feedback textarea to clarify why it isn't possible to further refine their offer preferences. You can learn from this feedback and implement more/better preferences. Also, you can add a checkbox to "never show this again".

**Comment by bartervg** [comment]
How would this textarea differ from the offer comments? If offer comments are disabled, that's the sender/receiver's decision. In addition, there are notes (visible to the note taker only), and feedback (visible to mediators only).

**Comment by Revadike** [comment]
The point of this issue is to improve offer preferences for users. The only reason I suggested users for leaving feedback, is so YOU can read it can learn why offers ignoring offer preferences (set or not) are declined.

**Comment by bartervg** [comment]
Would this be visible to anyone else?

They could use 
![image](https://user-images.githubusercontent.com/28576132/105909587-ba2bf000-5ff5-11eb-8fbb-55b7e6dc6997.png) 
although this is ignored for the most part or they could mention the problem on the forum or Discord. 

They could also write it in the comment, then flag the comment (an abuse of the reporting system, but they would increase the likelihood of the comment being seen, although risks getting warned if the feedback isn't constructive).

**Comment by Revadike** [comment]
Imagine this example:
*action* Offer declined
*prompt* Why did you decline this offer?
*action* User chooses option: I'd like to receive offers for wishlist only
*prompt* Please review your offer preferences

**Comment by bartervg** [comment]
AI support chatbot, useful? sure; feasible, unlikely. Unless it responds with `Please review your offer preferences` every time.

**Comment by Revadike** [comment]
I just prefer offers preferences were better fine-tuned for more people, so no time will be wasted by useless offers, or worse, growing ignore lists.

**Comment by bartervg** [comment]
That makes sense, but rather than another text input, some sort of protip section about how to provide better feedback and fine tune offer preference to reduce unwanted offers.

**Comment by Revadike** [comment]
Yes, this is what I mean!

**Comment by bartervg** [comment]
Step 1: better offer preference
This needs to be better before protips can be dispensed. I know it's been `soon` for too long.

**Comment by Revadike** [comment]
> Step 1: better offer preference
> This needs to be better before protips can be dispensed. I know it's been `soon` for too long.

Yes, and this is the reason why I suggested the text feedback. For you to learn what users would like to see in offer preferences.
